samples/fr-platform/amster/config/realms/root/BaseUrlSource.json

```diff
@@ -0,0 +1,19 @@
+{
+  "metadata" : {
+    "realm" : "/",
+    "amsterVersion" : "6.5.0-M11",
+    "entityType" : "BaseUrlSource",
+    "entityId" : "BaseUrlSource",
+    "pathParams" : { }
+  },
+  "data" : {
+    "_id" : "",
+    "fixedValue" : "https://login.sample.svc.cluster.local",
```

Comment:
we should figure out how to get rid of this hard coded value. I think commons could be used here?
 &{fqdn}